--- a/node_modules/@strapi/content-manager/dist/admin/hooks/useDocumentActions.mjs
+++ b/node_modules/@strapi/content-manager/dist/admin/hooks/useDocumentActions.mjs
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { useNotification, useTracking, useAPIErrorHandler } from '@strapi/admin/strapi-admin';
-import { useGetAIFeatureConfigQuery, useAIAvailability } from '@strapi/admin/strapi-admin/ee';
+import { useAIAvailability } from '@strapi/admin/strapi-admin/ee';
 import { useIntl } from 'react-intl';
 import { useNavigate } from 'react-router-dom';
 import { useRelationModal } from '../pages/EditView/components/FormInputs/Relations/RelationModal.mjs';
